public/config.sample.json

```diff
@@ -9,6 +9,8 @@
   "googleAnalyticsId": "UA-12345678-9",
   "requireLogin": false,
   "basicAuthorization": "",
+  "authClientId": "example_client_id",
+  "authAuthority": "${authServerHost}/connect",
```

Comment:
just using it to get the point across. Would `"authAuthority": "<your_host>/connect" be clearer?